# Originally from:
# https://build.pub.meego.com/package/files?package=qttools&project=home%3Abostik%3AQt5a1
# Added -Wl,--no-as-needed for Debian/Ubuntu
diff --git a/src/linguist/lrelease/lrelease.pro b/src/linguist/lrelease/lrelease.pro
index 792bc99..413e322 100644
--- a/src/linguist/lrelease/lrelease.pro
+++ b/src/linguist/lrelease/lrelease.pro
@@ -12,3 +12,7 @@ qmake.value = $$shell_path($$QMAKE_QMAKE)
 QT_TOOL_ENV += qmake
 
 load(qt_tool)
+
+# When linking with libQt5Boostrap.a we need zlib too
+LIBS += -Wl,--no-as-needed
+LIBS += -lz
